commands/utils/validate.js

```diff
@@ -51,6 +51,18 @@ module.exports = validate_config = function (lt_config, validation_configs) {
       reject("Error!! Parallels value not correct");
     }
 
+    //validate max duration
+    if ("max_duration" in lt_config["run_settings"]){
```

Comment:
add validation that it should be a int and throw an error  if that fails , here error is thrown only in this condition
```parseFloat(max_duration) <2 || parseFloat(max_duration) > 240```

Reply:
it is kept float intentional, as we want float number also. we are converting it to floor value in liisa.